test/passes/optimize-instructions_all-features.wast

```diff
@@ -3093,6 +3093,98 @@
     )
     (unreachable)
   )
+  (func $fdiv-32-power-2 (param $x f32)
```

Comment:
Can you add tests for NaNs and Infinities as well?

Reply:
Done